### PR TITLE
Rm mvp triton notification service

### PR DESCRIPTION
**Pull request by samouzegar10**
the notification service has been set to run every hour, let me know if there is anything that is to be changed in the code. thank you

### CODE REVIEW

**Comment by samouzegar10** [comment]
!!!!!! IMPORTANT ~~~~~ please merge the interactive guide pr first

File: triton-types/models/freezeman.d.ts [hunk 1]
```diff
@@ -12,11 +12,14 @@ export type ProjectID = number
...
     readonly id: DatasetID
     readonly external_project_id: string
+    readonly project_name: string
     readonly run_name: string
     readonly lane: number
     readonly files: ReadonlyArray<DatasetFile["id"]>
     readonly readset_count: number
     readonly released_status_count: number
+    readonly blocked_status_count: number
+    readonly metric_report_url?: string
```
Comment (nafiz1001) [inline triton-types/models/freezeman.d.ts:22]: If you wish to keep `metric_report_url`, it's best to leave it commented out with reason provided

**Comment by samouzegar10** [comment]
this branch has been manually pushed unto main

File: triton-server/package.json
Comment (nafiz1001) [inline triton-server/package.json:1]: i don't think you're using any new package

Comment (samouzegar10) [reply triton-server/package.json:1]: for the test email yea

Comment (samouzegar10) [reply triton-server/package.json:1]: and for cron as well

**Pull request merged**